packages/hoppscotch-selfhost-web/src/platform/auth/auth.platform.ts

```diff
@@ -317,9 +318,20 @@ export const def: AuthPlatformDef = {
   async setEmailAddress(_email: string) {
     return
   },
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+
   async setDisplayName(name: string) {
-    return
+    if (!name) return E.left("USER_NAME_CANNOT_BE_EMPTY")
+
+    const res = await updateUserDisplayName(name)
+
+    if (E.isRight(res)) {
+      probableUser$.next({
```

Comment:
Here we're only updating the value of probableUser. this will lead to currentUser and probableUser having different displayName after the update. we could use `setUser` function which sets both probableUser and currentUser instead of updating probableUser only.